javascript/README-NATIVE.md

```
@@ -0,0 +1,415 @@
+
+##### `listProxyTunnels()`
+
+List all active proxy tunnels.
+
+```typescript
+const tunnels = await client.listProxyTunnels();
+```
+
+### Gitea Methods
+
+##### `getGiteaCredentials()`
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)